src/web/components/KeyPairs/KeyPairsTable.scss

```diff
@@ -0,0 +1,65 @@
+.key-pairs {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+
+  .key-pairs-table {
+    margin-top: 28px;
+    font-size: 0.875rem;
+
+    thead {
+      font-weight: 700;
+      border: none;
+    }
+
+    .description {
+      width: 30%;
```

Comment:
I think the percentage in total is over 100%, maybe some of the width here is redundant?

Reply:
I set the public key column width to 40% and let the browser handle the others.  I think what we show on this grid may change soon, so we can revisit it.